internal/controllers/core/extension/reconciler.go

```diff
@@ -106,19 +110,31 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return ctrl.Result{}, err
 	}
 
+	if changed {
+		repoType := "http"
+		if strings.HasPrefix(repo.Spec.URL, "file://") {
+			repoType = "file"
+		}
+		r.analytics.Incr("api.extension.load", map[string]string{
+			"ext_path":      ext.Spec.RepoPath,
+			"repo_url_hash": analytics.HashSHA1(repo.Spec.URL),
+			"repo_type":     repoType,
+		})
+	}
+
 	return ctrl.Result{}, nil
 }
 
-// Generic status update.
-func (r *Reconciler) updateStatus(ctx context.Context, ext *v1alpha1.Extension, mutateFn func(*v1alpha1.ExtensionStatus)) (*v1alpha1.Extension, error) {
+// Generic status update. Returns true if the server changed.
+func (r *Reconciler) updateStatus(ctx context.Context, ext *v1alpha1.Extension, mutateFn func(*v1alpha1.ExtensionStatus)) (*v1alpha1.Extension, bool, error) {
 	update := ext.DeepCopy()
 	mutateFn(&(update.Status))
 
 	if apicmp.DeepEqual(update.Status, ext.Status) {
-		return update, nil
+		return update, false, nil
 	}
 	err := r.ctrlClient.Status().Update(ctx, update)
-	return update, err
+	return update, true, err
```

Comment:
I feeeeel like this should only return `true` if `err == nil` but it doesn't actually matter since we always early return in that case

Reply:
done!